# Homework #6

**See Canvas for the HW #6 assignment and due date**. Complete all of the following problems. Ideally, the theoretical problems should be answered in a Markdown cell directly underneath the question. If you don't know LaTex/Markdown, you may submit separate handwritten solutions to the theoretical problems, but please see the [class scanning policy](https://docs.google.com/document/d/17y5ksolrn2rEuXYBv_3HeZhkPbYwt48UojNT1OvcB_w/edit?usp=sharing). Please do not turn in messy work. Computational problems should be completed in this notebook (using the R kernel). Computational questions may require code, plots, analysis, interpretation, etc. Working in small groups is allowed, but it is important that you make an effort to master the material and hand in your own work. 



## A. Theoretical Problems

## A.1 [10 points] Approximate Confidence Interval for Proportions

Recall from an earlier assignment that if $np > 5$ and $n(1-p) > 5$, then $X \sim \text{Bin}(n,p)$ is well-approximated by $Y \sim N\left(np, np(1-p)\right)$. Use this approximation in the question below. In particular, an approximate $(1-\alpha)\times 100\%$ confidence interval for a population proportion $p$ is given by

\begin{align*}
\widehat{p} \pm z_{1-\alpha/2}\sqrt{\frac{\widehat{p}(1-\widehat{p})}{n}},
\end{align*}

where $\widehat{p}$ is the sample proportion. 

**Suppose that 12 people in a sample of 95 are members of the Green Party. Calculate an approximate 90% confidence interval for the true proportion of Green Party members in the population. Interpret this interval.**

## A.2 [10 points] Speed of Light

In 1881 Michelson and Newcomb measured the time light took to travel a distance of $7400$ meters. From a study of their experimental setup and a descriptive study of their $64$ measurements, we conclude that the data can be assumed to be i.i.d. These measurements yield the following sample quantities in microseconds for the sample mean $\bar x$ and sample standard deviation $s$:
$$\bar{x}=27.75, s=5.08$$
Construct a 95% confidence interval for the time light takes to travel $7400$ meters.



## A.3 A Change in Confidence

A journal article reports that a sample of size $n = 5$ was used as a basis for calculating a $95\%$ CI for the true average natural frequency (Hz) of delaminated beams of a certain type. The resulting interval was $(229.764, 233.504)$. You decide that a confidence level of 99% is more appropriate than the 95% level used.

**A.3 [14 points] (a) What are the limits of the 99% interval?**


## A.4 MLEs

Suppose that $X_1,...,X_n \overset{iid}{\sim}N(\mu, \sigma^2)$, where $\sigma$ is known, and we are ultimately interested in an estimator for $\theta = \mu^2$.

**A.4 (a) [12 points] First, find the MLE of $\mu$.**

**A.4 (b) [4 points] Find the maximum likelihood estimator (MLE) for $\theta$, denoted $\widehat{\theta}$.**

This should be easy!

**A.4(c) [10 points] Compute the bias of $\widehat{\theta}$, denoted $\text{Bias}(\widehat{\theta})$. Recall that $Bias(\widehat{\theta}) = E(\widehat{\theta}) - \theta$.**

## B. Computational Problems

## B.1 Hubble Data

Load `hubble.csv` into `R`. A description of the variables can be obtained from page 73 of https://cran.r-project.org/web/packages/gamair/gamair.pdf.

In [7]:
data = read.csv("hubble.csv",header=TRUE)
head(data)
length(data[,1])

X Galaxy   y    x    
1 1 NGC0300   133  2.00
2 2 NGC0925   664  9.16
3 3 NGC1326A 1794 16.14
4 4 NGC1365  1594 17.95
5 5 NGC1425  1473 21.88
6 6 NGC2403   278  3.22

[1] 24

**B.1 (a) [20 points] Calculate the $85\%$ confidence interval for the mean of a galaxy's distance from Earth in megaparsecs in `R` by doing the computation explicitly.**

In [9]:
# Here the size of n is less than 30 (ie. 24), so we will use the t-distribution 
alpha = 0.15
n = length(data$x)
s = sd(data$x)
xbar = mean(data$x)
t_value = qt(1-alpha/2,n-1)

lower_CI = xbar - t_value * (s/sqrt(n))
upper_CI = xbar + t_value * (s/sqrt(n))

lower_CI
upper_CI


[1] 10.28695

[1] 13.82222

**B.1 (b) [10 points] Find a built-in `R` function that does this computation automatically and verify that the built in `R` function does the same thing as the confidence interval formula used in part (a).**

In [11]:
t_test_R = t.test(data$x, conf.level = 0.85)
t_test_R$conf.int

[1] 10.28695 13.82222
attr(,"conf.level")
[1] 0.85

The calculation by using the Built in function and the calculation by using the formula both gave the same result.

**B.1(c) [10 points] Interpret the confidence interval.**

- We are 85% confident that the true mean of a galaxy's distance from the earth lies within the range of 10.2869 - 13.82222.
- In repeated sampling, 85% of all Confidence Intervals obtained from the sampling will contain the true population mean of a galaxy's distance from the earth.
- It does not mean that there is an 85% probability that the true population mean lies in this interval.